mirbot1:

laser_merger:
  ros__parameters:
    laser_1_topic: f_scan
    laser_2_topic: b_scan
    merged_scan_topic: scan

    target_frame: robot2/base_link

    laser_1_x_offset: 0.0
    laser_1_y_offset: 0.0
    laser_1_yaw_offset: 0.0

    laser_2_x_offset: 0.0
    laser_2_y_offset: 0.0
    laser_2_yaw_offset: 0.0

    tolerance: 0.01
    queue_size: 5

    angle_increment: 0.001
    scan_time: 0.067

    range_min: 0.05
    range_max: 10.0

    min_height: -1.0
    max_height: 1.0

    angle_min: -2.351592654
    angle_max: 2.351592654

    inf_epsilon: 1.0
    use_inf: false

    allowed_radius: 0.45
    enable_shadow_filter: true
    enable_average_filter: true


